Dockerfile

```
@@ -94,9 +94,10 @@ USER appuser
 # Expose the application port
 EXPOSE 4444
 
-# Health check endpoint
+# Health check endpoint. Hits /health (not /) and asserts the reported status so
+# a server that answers but cannot describe its own state is treated as failing.
 HEALTHCHECK --interval=30s --timeout=3s --start-period=40s --retries=3 \
```

Hadolint (2.15.1)

[warning] Use arguments JSON notation for CMD and ENTRYPOINT arguments

(DL3025)